src/main/java/com/patchworkmc/manifest/converter/ModManifestConverter.java

```diff
@@ -105,4 +110,26 @@ private static JsonArray convertAuthorsList(String authors) {
 
 		return array;
 	}
+
+	private static JsonObject getDependencies(ModManifest manifest, ModManifestEntry mod) {
+		JsonObject depends = new JsonObject();
+		Map<String, List<ModManifestDependency>> dependencyMap = manifest.getDependencyMapping();
+		dependencyMap.get(mod.getModId()).forEach(c -> {
+			if (c.isMandatory()) {
+				depends.addProperty(c.getModId(), c.getVersionRange());
+			}
+		});
+		return depends;
+	}
+
+	private static JsonObject getRecommendedDependencies(ModManifest manifest, ModManifestEntry mod) {
+		JsonObject recommends = new JsonObject();
+		Map<String, List<ModManifestDependency>> dependencyMap = manifest.getDependencyMapping();
+		dependencyMap.get(mod.getModId()).forEach(c -> {
+			if (!c.isMandatory()) {
+				recommends.addProperty(c.getModId(), c.getVersionRange());
```

Comment:
Is *recommends* proper here?  *suggests* seems more appropriate.

Reply:
Forge doesn't make a distinction, so I figured strongest option would be the best. I can change it if you want, though.

Reply:
I think suggests is closer to what we want, because generally `after` dependencies are used for inter mod compat just for if the mod is loaded.